shell/packages/sandstorm-db/db.js

```diff
@@ -56,6 +55,11 @@ if (Meteor.isServer && process.env.LOG_MONGO_QUERIES) {
 //       verifiedEmail: Only provided by some services. Cannot be directly edited by the user.
 //       main: True if this is the user's main identity.
 //       noLogin: True if the user does not trust this identity for account authentication.
+//       devName: Only present when `service` is "dev". A string containing the name used to create
+//                this dev account.
+//       emailTokens: Only present when `service` is "emailToken". An array of objects of the form
```

Comment:
Ah, maybe it's not so bad. We could do something like:

```
db.users.find({}, {"identities.service.emailToken._dummy": 1, "identities.service.dev._dummy": 1})
```

where the `_dummy` field need not exist in the actual data.